runtime/js-compute-runtime/builtins/request-response.cpp

```diff
@@ -29,8 +29,68 @@
 namespace builtins {
 
 namespace {
+bool error_stream_controller_with_pending_exception(JSContext *cx, JS::HandleObject controller) {
```

Comment:
This used to be within event_loop.cpp but I moved it into request-response as it is only used on requests